### PR TITLE
chore: onboard secure notebook blueprint

### PR DESCRIPTION
**Pull request by gtsorbo**
legacy blueprint for AI notebook security is due for an update, and was never onboarded to CI properly

### CODE REVIEW

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -663,5 +663,12 @@ locals {
       owners      = ["Saipriyavk", "ChrisBarefoot"]
       topics      = local.common_topics.net
     },
+    {
+      name        = "notebooks-blueprint-security"
```
Comment (g-awmalik) [inline infra/terraform/test-org/org/locals.tf:667]: Any reason this is not prefixed with "terraform"?

Comment (gtsorbo) [reply infra/terraform/test-org/org/locals.tf:667]: @g-awmalik This was an older blueprint and so it didn't follow standard naming - open to renaming the repo itself if that's possible

Comment (g-awmalik) [reply infra/terraform/test-org/org/locals.tf:667]: Sorry for the delaying response. Let's let this one slide.

**Review by g-awmalik — APPROVED** [review]

**Review by g-awmalik — CHANGES_REQUESTED** [review]
@gtsorbo - I tried applying but the name is too long. It's already 28 chars and with a prefix of "ci-" it becomes 31...should be less than 30. Can you rename this or provide an alternate `short_name` for this?

**Review by g-awmalik — APPROVED** [review]

**Comment by g-awmalik** [comment]
Applied.

**Pull request merged**